### ISSUE TITLE
feature suggestion: could you please add capability of virtual ribbon menu for nvda users?

### ISSUE THREAD

**Issue by zahra21** [nvaccess/nvda#7061]
hello every one.
jaws12 and later versions have the feature called virtual ribbon menu.
when jaws user check this option, he-she can use ribbon menu like classic menu by using arrow keys!
in this case, the ribbon menu are remain ribbon, but jaws users can navigate the menus like classic menus by just using arrow keys!
i know some persons that want use nvda instead of jaws, but dont use nvda and stick to jaws only because of missing this feature in nvda.
did someone request this feature previously?
do you have any plan to include this feature in nvda 2017.2 and newer versions?
nvda developers and programmers,
i realy appreciate you if you tell me your plan about virtual ribbon in the future versions of nvda?
do you work on it?
if so, did you start this work previously?
or in which version of nvda i should expect this feature?
thanks extremely for your great screen reader nvda, God bless you all and i pray for you in my daily five times obligotary prayers and request divine infinite mercy and blessings for you.

**Comment by Qchristensen** [comment]
Thanks for your suggestion!  Much as I agree regarding the benefits of a traditional style menu, we hadn't considered adding this feature to NVDA.  Most people find that the ribbon isnt too daunting once you get the hang of it.  Here are some tips:

- Press ALT to access the ribbon, then left and right ARROWS to move between the different ribbons, as with menus.
From here:

- Press CONTROL+LEFT ARROW or CONTROL+RIGHT ARROW to move between groupings on the current ribbon, these are the main groups of items, eg Font, Paragraph and styles are some of the groupings on the Home ribbon of Microsoft Word.
- Press TAB to move through the individual options in the current group (the focus will move to the next group automatically).
- Press ENTER to activate an item.  Some items have menus inside them.  In this case press the down arrow to move through these and then ENTER again to activate a specific item.

As you select things, the shortcut keys to jump directly to that object are also read, so you can quickly access that feature again if needed. For instance, press alt+n for the insert ribbon, then t for table, then use the arrow keys (table is one of the few options that uses both left / right and up / down arrows to select the appropriate option) to set the size, then press ENTER to insert it.

I'd strongly recommend the "Basic Training for NVDA" module, available from http://www.nvaccess.org/shop/  As well as use of the ribbon, it includes many other tips and tricks to get you using NVDA more efficiently (If you are interested in Microsoft Office, the "Microsoft Word with NVDA" module has specifics on using Microsoft Word.  An Excel module is due out later this year).

Kind regards

Quentin.

**Comment by jcsteh** [comment]
This is a duplicate of #4844. As discussed above, while the experience is different, it is quite possible to navigate ribbons using the keyboard. You can use object navigation as well. Also, doing this provides an entirely different user experience for screen reader users and such alternative interfaces are prone to break as changes are made in applications. As such, this is not something we are considering for implementation in NVDA.
